pkg/loop/internal/pb/keyvalue_store.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb";
+
+package loop;
+
+import "google/protobuf/empty.proto";
+
+message StoreKeyValueRequest {
+  string key = 1;
+  bytes value = 2;
+}
+
+message GetValueForKeyRequest {
+  string key = 1;
+}
+
+message GetValueForKeyResponse {
+  bytes value = 1;
+}
+
+service KeyValueStore {
+  rpc StoreKeyValue(StoreKeyValueRequest) returns (google.protobuf.Empty);
```

Comment:
@ettec Not super keep on the lack of symmetry here between the GRPC methods and the actual interface. Doubly so since you're using Store to mean two different things.

Could we either shorten or try to match the interface as closely as possible?

eg. Store(StoreRequest)
Get(GetRequest)

Bonus points would be to rename Store everywhere with Put, but that's maybe veering towards a stylistic preference.

Reply:
It was this originally but conflicted with other services 'GetRequest' message - hence the more verbose name

Reply:
@ettec Test